include/deal.II/grid/grid_generator.h

```diff
@@ -1903,7 +1901,8 @@ namespace GridGenerator
                        const Triangulation<dim, spacedim> &triangulation_2,
                        Triangulation<dim, spacedim> &      result,
                        const double duplicated_vertex_tolerance = 1.0e-12,
-                       const bool   copy_manifold_ids           = false);
+                       const bool   copy_manifold_ids           = false,
+                       const bool   copy_boundary_ids           = false);
```

Comment:
The cause of the failing test is that the new parameter is missing in the other `merge_triangulations()` instance.